NetworkingSampleApp/NetworkingSampleApp/Scenes/Users/UsersViewModel.swift

```diff
@@ -0,0 +1,82 @@
+//
+//  UsersViewModel.swift
+//  NetworkingSampleApp
+//
+//  Created by Matej Molnár on 07.12.2023.
+//
+
+import Foundation
+import Networking
+
+@MainActor
+final class UsersViewModel: ObservableObject {
```

Comment:
@matejmolnar @cejanen Why do we sometimes use a service as well in the VMs and sometimes we don't? 

Eg. the upload also has a `UploadService` 😅

Reply:
Usually for samples we don't. For samples we try to keep code as simple as possible just to showcase the library usage. Because upload service logic is shared in multiple VMs it made sense to extract it own service class.

Reply:
Although I am still not entirely sure why is the UploadAPIManager implementation so different and more complex than the DownloadAPIManager 🤔 but that's a topic for another debate 😄

Reply:
@matejmolnar I think that was my first comment and conclusion on the Upload PR, that it could have been almost 1:1 with the download manager.